### PR TITLE
Prism-Bukkit-[Improvement] Paste error handling and logging

### PR DESCRIPTION
**Pull request by Narimm**
This improves paste handling for debug
Including showing the deletion key if the paste is anon.

Signed-off-by: Narimm <benjicharlton@gmail.com>

### CODE REVIEW

File: src/main/java/me/botsko/prism/commands/DebugCommand.java [hunk 1]
```diff
@@ -20,6 +20,7 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.List;
+import java.util.function.Consumer;
```
Comment (Narimm) [inline src/main/java/me/botsko/prism/commands/DebugCommand.java:23]: Codacy Issue found: [Avoid unused imports such as 'java.util.function.Consumer'](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5499560)

**Comment by Narimm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5499560&bid=18024092)

File: src/main/java/me/botsko/prism/Prism.java [hunk 2]
```diff
@@ -344,9 +344,11 @@ public void onEnable() {
         }
         if (getConfig().getBoolean("prism.paste.enable")) {
             pasteKey = Prism.config.getString("prism.paste.api-key", "API KEY");
-            if (pasteKey != null && pasteKey.startsWith("API key")) {
+            if (pasteKey != null && pasteKey.startsWith("API key") || pasteKey.length() < 6) {
```
Comment (alchemistmatt) [inline src/main/java/me/botsko/prism/Prism.java:348]: Should this be `&& !pasteKey.startsWith("API key")` ?

**Review by alchemistmatt — COMMENTED** [review]
You might have missed a ! (aka not) in the if check; see Prism.java, line 347

File: src/main/java/me/botsko/prism/Prism.java (hunk 2 above)
Comment (Narimm) [reply src/main/java/me/botsko/prism/Prism.java:348]: nope thats the default...which we want to make null

Comment (Narimm) [reply src/main/java/me/botsko/prism/Prism.java:348]: basically that line filters out bad api keys sort of

**Pull request merged**